ota-plus-web/app/reactapp-new/src/App.tsx

```diff
@@ -35,9 +36,9 @@ export const App = (props: IProps) => {
         <AppNavbar />
         <AppMain>
           <Title size="large">Welcome</Title>
-          <Container>
-            <Title>Recently created</Title>
-          </Container>
+          <div style={{ width: 800 }}>
```

Comment:
Why inline style?

Reply:
this is just preview in app.tsx which will be removed, just wanted to have better visualization for testing